src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstancePropertyValue.scala

```diff
@@ -279,13 +279,9 @@ object InstancePropertyValue {
       case Float32List(values) => Json.arr(values = values.map(Json.fromFloatOrString): _*)
       case Float64List(values) => Json.arr(values = values.map(Json.fromDoubleOrString): _*)
       case DateList(values) =>
-        Json.arr(values =
-          values.map(d => Json.fromString(d.format(InstancePropertyValue.Date.formatter))): _*
-        )
+        Json.arr(values = values.map(d => Json.fromString(d.format(Date.formatter))): _*)
       case TimestampList(values) =>
-        Json.arr(values =
-          values.map(d => Json.fromString(d.format(InstancePropertyValue.Timestamp.formatter))): _*
-        )
+        Json.arr(values = values.map(d => Json.fromString(d.format(Timestamp.formatter))): _*)
```

Comment:
that one is annoying, scalafmt (by default the 2.13 one?) inserts newlines and that seems to break scala 3 as it doesn't recognize `: _*` in multiline single-argument version..
But otoh removing `InstancePropertyValue.` prefix looks fine so let's do that :+1: